site/tryit/README.md

```diff
@@ -80,32 +94,36 @@ docker run --rm sorintlab/agolademo --token "admintoken" --gateway-url http://17
 --skip-ssh-host-key-check
 ```
 
-"admintoken" is a token defined in the default agolademo configuration and will let you act with the api as an admin without the need of an user created inside agola.
+* `--token "admintoken" ` is a token defined in the default agolademo configuration and will let you act with the api as an admin without the need of an user created inside agola.
+
+* `--skip-ssh-host-key-check` is used to speed up things and tells agola to not check gitea host ssh host key when cloning repositories. The right thing to do will be to provide the ssh host key using the option `--ssh-host-key`. 
 
-* `--skip-ssh-host-key-check` is used to speed up things and tells agola to not check gitea host ssh host key when cloning repositories. The right thing to do will be to provide the ssh host key using the option `--ssh-host-key`. You can get the host key using `ssh-keyscan $giteahost` and choosing the ecdsa or rsa host key provided line (use the whole line)
+You can get the host key using `ssh-keyscan $giteahost` and choosing the ecdsa or rsa host key provided line (use the whole line)
 
 ### Register
 
 Login to the agola web ui on http://172.17.0.1:8000 and choose register, then **register with gitea**. If everything goes well gitea will ask you to accept the authorization and then you'll be redirected back to the user registration form. Next you can Login.
 
 ### Create a user api token
 
-Use the web interface or the cli:
+Use the ***web interface*** or the ***cli*** :
 
 ```
 docker run --rm sorintlab/agolademo --token admintoken --gateway-url http://172.17.0.1:8000 user token create -n $YOUR_AGOLA_USERNAME -t default
 ```
-
-Save the token since it won't be displayed again.
+::: warning
+Save the **token** since it won't be displayed again.
+:::
 
 #### Testing with an example repository
 
 We'll use the [agola-example-go](https://github.com/agola-io/agola-example-go) repository
-* Clone to above repository locally
 
-* Create a repository on gitea called `agola-example-go`
+1. Clone to above repository locally
+
```

Comment:
Do not add an empty line between numbered lists (and you could just use `1.` for every item and they will be automatically numbered)